locale/en/submission.po

```diff
@@ -2189,6 +2189,14 @@ msgstr "Before you begin"
 msgid "submission.wizard.categories.description"
 msgstr "Select only the categories that are appropriate for your submission."
 
+msgid "submission.wizard.categories.descriptionWithCircularReferenceWarning"
+msgstr ""
+"Select only the categories that are appropriate for your submission.<br>"
+"<sub class=\"text-negative\">Some categories are not displayed due to circular references. Please contact your Admin to resolve this.</sub>"
```

Comment:
For this and the one below, I would add some additional context, something like "due to circular references in the category hierarchy." I would also say "Site Administrator" rather than "Admin" as this info will be listed on the site with contact info.